reactioncommerce/reaction
# 2422 Fix on email invite link leading to blank screen

Resolves #2422 

**Testing**

- Clone this branch
- Login as admin
- Invite user with valid email
- Open invitation email and visit link
- Observe that the `Update Password` dialog is not broken
